gobblin-metrics/src/main/java/gobblin/metrics/MetricContext.java

```diff
@@ -44,8 +52,13 @@
 import com.google.common.collect.ImmutableSortedSet;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
 import com.google.common.io.Closer;
 
+import gobblin.metrics.notify.EventNotification;
```

Comment:
Can you rename `notify` to `notification`? Although it is verbose, it's better than using a verb as a package name.